api/src/pcapi/core/providers/allocine_venue.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+from typing import cast
+
+from pcapi.core.offerers.models import Venue
+import pcapi.core.providers.exceptions as providers_exceptions
+from pcapi.core.providers.models import AllocinePivot
+from pcapi.core.providers.models import AllocineTheater
+from pcapi.core.providers.repository import get_allocine_pivot
+from pcapi.core.providers.repository import get_allocine_theater
+
+
+class AlloCineVenue:
+    def __init__(self, venue: Venue):
+        self.allocine_pivot = get_allocine_pivot(venue)
+        self.allocine_theater = get_allocine_theater(venue)
+        if not self.has_pivot() and not self.has_theater():
+            raise providers_exceptions.UnknownVenueToAlloCine()
+
+    allocine_pivot: Optional[AllocinePivot]
+    allocine_theater: Optional[AllocineTheater]
+
+    def has_pivot(self) -> bool:
+        return isinstance(self.allocine_pivot, AllocinePivot)
```

Comment:
Quelle est la différence avec `return allocine_pivot is not None` ? (pareil en-dessous)

Reply:
Rien dans le résultat.
Si tu poses la question, c'est que tu préfères le `is not None`, je change.

Reply:
Bien vu, je préfère et c'est plus lisible ; avec isinstance ça suggère que ça peut être de plusieurs types de classes, et qu'on vérifie donc le type